### DISCUSSION TITLE
Updating API support coverage tracking issues

### DISCUSSION THREAD

**Discussion by Diatonator** [ruffle-rs/ruffle#2314] [General]
I see a few things which got implemented over time, but the tracking issues still show those things as not yet implemented. 

https://github.com/ruffle-rs/ruffle/pull/2309
https://github.com/ruffle-rs/ruffle/pull/2238
https://github.com/ruffle-rs/ruffle/pull/2217
https://github.com/ruffle-rs/ruffle/pull/2190

Do you have a certain set period when those API tracking issues get updated? Or you just forget to do this at times?
Could be useful to have the tracking issues be reasonably up-to-date, as it'll likely help to understand for some issues if they are bugs or not-yet-implemented things (F12 console doesn't speak for all of the unimplemented things unfortunately, at least for AVM1). 
Currently the up-to-date alternative is searching through the current issues and closed/open PRs. Or logging the issue and getting the answer from contributors.
Maybe good idea to allow contributors to somehow change the tracking issues as they implement things, or some automation?

**Comment by Diatonator** [comment]
there are of course merged PR mentions which go after those issues' main posts, but any unified format of showing the info is better anyway

**Comment by ROBERT-MCDOWELL** [comment]
maybe make a script in cron that load a file from github and update the tracking table automatically with a normalized line information?
